### PR TITLE
Wip sgh polish

### PR DESCRIPTION
**Pull request by sgharms**

### CODE REVIEW

**Comment by sgharms** [comment]
@drakeltheryuujin Please review these edits. I've added some comments here for some follow-on work and some points update your process on.

File: README.md [hunk 1]
```diff
@@ -52,128 +52,115 @@ up IRB by opening your terminal; type `irb` and hit enter and type the commands
...
-If everything was typed correctly, your results should be "100" and "-121".
+If everything was typed correctly, your results should be `100` and `-121`.
 Nothing out of the ordinary, right?
 
 
 ## Recognize `/` as the Symbol For Division
 
-The symbol for the division (`/`) is a forward slash. This may differ from
-writing common arithmetic as well and is not the only difference you'll notice.
+The symbol for the division (`/`) is a forward slash. There's no handy `÷`
```
Comment (sgharms) [inline README.md]: The old text is waayyy too subtle about something that's verrrrryyyy important.

Can we add something about not doing that in our "how to draft" guide.

File: README.md [hunk 2]
```diff
@@ -52,128 +52,115 @@ up IRB by opening your terminal; type `irb` and hit enter and type the commands
...
 If you tested out each one, you might notice that when dividing a number that
 has a remainder, or fraction in the value, that amount is left out and the
 result you're left with is a whole number. This is different from regular
 arithmetic, and is a feature in programming that may feel unfamiliar.
 
-When you type an a statement like `9 / 2` into IRB, you would expect the result
-to be 4.5, but what is returned is actually 4. This is because of the type of
-number being used. In Ruby, and most programming languages, numbers can be
-_integers_, which are whole numbers, or _floats_, which are decimal numbers.
+When you type a statement like `9 / 2` into IRB, you would expect the result to
```
Comment (sgharms) [inline README.md]: ‼️ double check my logic here, but when i tried this out in IRB it worked the same for both float and integer division.

File: README.md [hunk 3]
```diff
@@ -52,128 +52,115 @@ up IRB by opening your terminal; type `irb` and hit enter and type the commands
...
-
-The remainder for this operation would be "2" if we could see the full number.
-If we perform the same operation using the modulo operator, our result that
-would also be "2".
-
-```ruby
-12 % 10
-# => 2
+5 % 2 #=> 1
+5.0 % 2 => 1.0
```
Comment (sgharms) [inline README.md]: see above.

File: README.md [hunk 4]
```diff
@@ -183,12 +170,16 @@ similar concept.
 ## Recognize `Math.sqrt`
 
 As mentioned previously, Ruby provides an extensive range of math related
-functions. It gives you access to mathematic tools such as square root, sine,
-cosine, and tangent via Ruby methods. To access these in your code, refer to the
-module `Math` and call the method on the module.
+functions. It gives you access to mathematics tools such as square root, sine,
+cosine, and tangent via Ruby methods. A full description of the `Math` module
```
Comment (sgharms) [inline README.md]: ‼️ this is a thing from Brian. We have to be clear when memorize versus refer.

File: README.md [hunk 5]
```diff
@@ -52,128 +52,115 @@ up IRB by opening your terminal; type `irb` and hit enter and type the commands
 below:
 
 ```ruby
-10 + 1
+10 + 1 #=> 11
```
[the remark this answers is not in the document]
Comment (drakeltheryuujin) [reply README.md]: @sgharms I wasn't sure if this should be the default, but I do agree this could be a standard.

File: README.md (hunk 1 above)
Comment (drakeltheryuujin) [reply README.md]: @sgharms I'll discuss with @jenmyers how we can word this for the guide

File: README.md (hunk 2 above)
Comment (drakeltheryuujin) [reply README.md]: @sgharms is there a setting that can be changed in IRB? `9 / 2` gives me `4` in my IRB.

File: README.md (hunk 3 above)
Comment (drakeltheryuujin) [reply README.md]: @sgharms Works for me as expected for this operation as well.

File: README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (sgharms) [reply README.md]: 👍

File: README.md [hunk 6]
```diff
@@ -52,128 +52,115 @@ up IRB by opening your terminal; type `irb` and hit enter and type the commands
...
+
+This is because of the _type_ of number being used. In Ruby, and most
+programming languages, numbers can be _integers_, which are whole numbers, or
+_floats_, which are decimal numbers.
+
 When you divide integers (whole numbers), they will return integers without
-rounding and the numbers behind the decimal is removed. If you type `9.0`,
-you're creating a float:
+rounding and the numbers behind the decimal is removed. When you divide floats,
+you'll get floats. If you mix integers with floats, you get a float.
```
Comment (sgharms) [inline README.md]: The old text implies, it seems to me that modulo only works on integers, but it works on integers and floats. That's what I meant to say (I tagged the wrong bloc of code).

File: README.md (hunk 3 above)
Comment (sgharms) [reply README.md]: See new comment, sorry for confusing.

File: README.md (hunk 4 above)
Comment (drakeltheryuujin) [reply README.md]: Got it!

File: README.md (hunk 6 above)
Comment (drakeltheryuujin) [reply README.md]: Makes sense

**Pull request merged**